libs/java/server_common/src/main/java/com/yahoo/athenz/common/ServerCommonConsts.java

```diff
@@ -23,6 +23,28 @@
 
     public static final String USER_DOMAIN_PREFIX = "user.";
 
+    // For CloudStore
+    public static final String ZTS_PROP_AWS_REGION_NAME                     = "athenz.zts.aws_region_name";
+    public static final String ZTS_PROP_AWS_CREDS_CACHE_TIMEOUT             = "athenz.zts.aws_creds_cache_timeout";
+    public static final String ZTS_PROP_AWS_CREDS_INVALID_CACHE_TIMEOUT     = "athenz.zts.aws_creds_invalid_cache_timeout";
+    public static final String ZTS_PROP_AWS_ENABLED                         = "athenz.zts.aws_enabled";
+    public static final String ZTS_PROP_AWS_CREDS_UPDATE_TIMEOUT            = "athenz.zts.aws_creds_update_timeout";
+    public static final String ZTS_SSH_TYPE                                 = "certtype";
+
+    // For S3ChangeLogStore
+    public static final String ZTS_PROP_AWS_BUCKET_NAME = "athenz.zts.aws_bucket_name";
+
+    // For ZMSFileChangeLogStore
+    public static final String ZTS_PROP_ZMS_URL_OVERRIDE    = "athenz.zts.zms_url";
```

Comment:
Moved ZTS consts here and fixed the references (which is the reason many files have changed)